nengo_gui/components/value.py

```diff
@@ -22,8 +21,15 @@ def __init__(self, obj):
         # the pending data to be sent to the client
         self.data = []
 
+        # grab the output of the object
+        self.output = obj
+        default_out = self.default_output(self.obj)
+        if(default_out):
+            self.output = default_out
+
+
         # the number of data values to send
-        self.n_lines = int(obj.size_out)
+        self.n_lines = self.output.size_out
```

Comment:
Why get rid of the ```int``` here?  That's needed because some versions of numpy give size_outs that can't be serialized.....